sql/changes/1.5/open_forms_callers.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+ALTER TABLE public.open_forms ALTER COLUMN id SET DEFAULT floor(random()*(1000000))+1;
+ALTER TABLE public.open_forms ADD COLUMN caller1 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN caller2 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN caller3 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN caller4 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN last_used timestamp without time zone;
```

Comment:
Do we want this in production? I mean, it's a great debugging tool, obviously; and caller1 to 5 make great input into a hash, but...